lib/zeebe/io_mapping.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Zeebe
+  class IoMapping
+    attr_accessor :inputs, :outputs
+
+    def initialize(moddle)
+      @inputs = moddle["inputParameters"]&.map { |parameter_moddle| Parameter.new(parameter_moddle) }
+      @outputs = moddle["outputParameters"]&.map { |parameter_moddle| Parameter.new(parameter_moddle) }
```

Comment:
Seems nice to just make these methods which default to `[]` like `Step#input_mappings`.